server/server.go

```diff
@@ -1,113 +1,52 @@
 package server
 
-import (
-	"path/filepath"
-
-	pb "github.com/kubernetes/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
-	"github.com/opencontainers/ocitools/generate"
-	"golang.org/x/net/context"
+const (
+	runtimeAPIVersion = "v1alpha1"
```

Comment:
I'm not sure what do you mean by API version - containers runtimes in OCI follow the "runtime-spec" which has an API but I think here we're talking about the CRI API version which isn't a concept tied to containers runtimes.